packages/commonwealth/server/migrations/20231212140801-divastaking-community-and-contract-update.js

```diff
@@ -0,0 +1,759 @@
+          },
+        ],
+        stateMutability: 'view',
+        type: 'function',
+      },
+      {
+        inputs: [
+          {
+            internalType: 'uint256',
+            name: 'proposalId',
+            type: 'uint256',
+          },
+        ],
+        name: 'state',
+        outputs: [
+          {
+            internalType: 'enum IGovernor.ProposalState',
+            name: '',
+            type: 'uint8',
+          },
+        ],
+        stateMutability: 'view',
+        type: 'function',
+      },
+      {
+        inputs: [
+          {
+            internalType: 'bytes4',
+            name: 'interfaceId',
+            type: 'bytes4',
+          },
+        ],
+        name: 'supportsInterface',
+        outputs: [
+          {
+            internalType: 'bool',
+            name: '',
+            type: 'bool',
+          },
+        ],
+        stateMutability: 'view',
+        type: 'function',
+      },
+      {
+        inputs: [],
+        name: 'version',
+        outputs: [
+          {
+            internalType: 'string',
+            name: '',
+            type: 'string',
+          },
+        ],
+        stateMutability: 'view',
+        type: 'function',
+      },
+      {
+        inputs: [],
+        name: 'votingDelay',
+        outputs: [
+          {
+            internalType: 'uint256',
+            name: '',
+            type: 'uint256',
+          },
+        ],
+        stateMutability: 'view',
+        type: 'function',
+      },
+      {
+        inputs: [],
+        name: 'votingPeriod',
+        outputs: [
+          {
+            internalType: 'uint256',
+            name: '',
+            type: 'uint256',
+          },
+        ],
+        stateMutability: 'view',
+        type: 'function',
+      },
+      {
+        inputs: [],
+        name: 'proposalThreshold',
+        outputs: [
+          {
+            internalType: 'uint256',
+            name: '',
+            type: 'uint256',
+          },
+        ],
+        stateMutability: 'view',
+        type: 'function',
+      },
+    ];
+    const abiData = JSON.stringify(abi);
+
+    const hashInstance = hasher({
+      coerce: true,
+      sort: true,
+      trim: true,
+      alg: 'sha256',
+      enc: 'hex',
+    });
+
+    const abiHash = hashInstance.hash(JSON.parse(abiData));
+    await queryInterface.sequelize.query(
+      `
+      INSERT INTO "ContractAbis" ("abi", "abi_hash", "verified", "nickname", "created_at", "updated_at")
+      VALUES (:abi, :abiHash, :verified, :nickname, NOW(), NOW())
+    `,
+      {
+        replacements: {
+          abi: abiData,
+          abiHash: abiHash,
+          verified: true,
+          nickname: 'GovernorCountingSimple',
+        },
+        type: Sequelize.QueryTypes.INSERT,
+        transaction,
+      },
+    );
+
+    // Retrieve the ID of the newly inserted "ContractAbi" (if needed for further operations)
+    const contractAbiId = await queryInterface.sequelize.query(
```

Comment:
Any reason we can't build this into the below update query instead? 

You can also return an ID from the above insert instead and use like this to avoid the query in both cases

Reply:
Merged into below query instead of having two seperate